lib/Runtime/Library/WebAssemblyInstance.cpp

```diff
@@ -368,13 +367,17 @@ void WebAssemblyInstance::LoadImports(
             {
                 if (!WebAssemblyTable::Is(prop))
                 {
-                    JavascriptError::ThrowWebAssemblyLinkError(ctx, WASMERR_NeedTableObject);
+                    throw Wasm::WasmCompilationException(_u("Invalid import '%s.%s'. Expected a WebAssembly.Table"), import->modName, import->importName);
```

Comment:
>Invalid import '%s.%s'. Expected a WebAssembly.Table [](start = 61, length = 52)

Can these error strings be centralized at one place? Can't we use RT_ERROR_MSG macros here?